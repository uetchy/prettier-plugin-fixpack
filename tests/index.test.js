const prettier = require("prettier");

const fixture = `
{
  "author": "",
  "version": "0.0.0",
  "description": "Pretiter plugin",
  "name": "prettier-plugin-fixpack",
  "main": "lib/index.js",
  "scripts": {
    "test": "jest"
  },
  "files": [
    "lib"
  ],
  "dependencies": {
    "json5": "^2.1.3",
    "find-up": "^5.0.0"
  },
  "peerDependencies": {
    "prettier": "^2.1.1"
  },
  "devDependencies": {
    "jest": "^26.4.2",
    "prettier": "^2.1.1",
    "release-it": "^13.6.6",
    "@release-it/conventional-changelog": "^1.1.4"
  },
  "homepage": "https://github.com/uetchy/prettier-plugin-fixpack",
  "repository": {
    "url": "https://github.com/uetchy/prettier-plugin-fixpack.git",
    "type": "git",
  },
  "license": "Apache-2.0",
  "bugs": {
    "url": "https://github.com/uetchy/prettier-plugin-fixpack/issues"
  },
  "keywords": [
    "prettier-plugin",
    "prettier",
    "package.json",
  ],
  "b": 1,
  "a": 1,
  "engines": {
    "node": ">= 12.18.3"
  },
}
`;

it("sort", () => {
  expect(
    prettier.format(fixture, {
      filepath: "package.json",
      plugins: ["."],
    })
  ).toMatchSnapshot();
});
