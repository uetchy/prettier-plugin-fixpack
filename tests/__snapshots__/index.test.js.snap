// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sort 1`] = `
"{
  \\"name\\": \\"prettier-plugin-fixpack\\",
  \\"description\\": \\"Pretiter plugin\\",
  \\"version\\": \\"0.0.0\\",
  \\"author\\": \\"\\",
  \\"scripts\\": {
    \\"test\\": \\"jest\\"
  },
  \\"main\\": \\"lib/index.js\\",
  \\"files\\": [
    \\"lib\\"
  ],
  \\"dependencies\\": {
    \\"find-up\\": \\"^5.0.0\\",
    \\"json5\\": \\"^2.1.3\\"
  },
  \\"peerDependencies\\": {
    \\"prettier\\": \\"^2.1.1\\"
  },
  \\"devDependencies\\": {
    \\"@release-it/conventional-changelog\\": \\"^1.1.4\\",
    \\"jest\\": \\"^26.4.2\\",
    \\"prettier\\": \\"^2.1.1\\",
    \\"release-it\\": \\"^13.6.6\\"
  },
  \\"homepage\\": \\"https://github.com/uetchy/prettier-plugin-fixpack\\",
  \\"repository\\": {
    \\"type\\": \\"git\\",
    \\"url\\": \\"https://github.com/uetchy/prettier-plugin-fixpack.git\\"
  },
  \\"bugs\\": {
    \\"url\\": \\"https://github.com/uetchy/prettier-plugin-fixpack/issues\\"
  },
  \\"license\\": \\"Apache-2.0\\",
  \\"keywords\\": [
    \\"package.json\\",
    \\"prettier\\",
    \\"prettier-plugin\\"
  ],
  \\"a\\": 1,
  \\"b\\": 1,
  \\"engines\\": {
    \\"node\\": \\">= 12.18.3\\"
  }
}
"
`;
